ia/src/load.py

```
@@ -0,0 +1,269 @@
+        send_message(client, f"{command} {args}")
+    elif command == utils.FORWARD:
+        if client.last_look[utils.PLAYER_CELL].count("player") > utils.CANT_MOVE and client.help_status:
+            return
+        send_message(client, command)
+    else:
+        send_message(client, command)
+    if len(client.commands) < utils.MAX_COMMANDS:
+        client.commands.append(command)
+
+
+def handle_client(client) -> None:
```

Ruff (0.11.9)

`handle_client` is too complex (15 > 10)

(C901)